_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,194 @@
+
+### New volkswagen van
+
+The CARLA garage has built an exciting new vehicle, the __Volkswagen T2 2021__. Add some charm to your city traffic with this beautifully retro styled vehicle. 
+
+![vwvan](/img/posts/2021-11-07/vwvan.gif){:class="img-fluid rounded mx-auto d-block"}
+
+### Opening Doors
+
+All generation 2 vehicles now have articulated doors that can be opened and closed through functions in the python API. This functionality is ideal for simulating scenarios in busy city streets and car parks where doors may open unpredictably while driving past or might be blocking access to parking spaces or passageways. 
+
+![doors](/img/posts/2021-11-07/doors.gif){:class="img-fluid rounded mx-auto d-block"}
+
+## Improved traffic manager logic
+
+The traffic manager has undergone a major update with updated logic for handling vehicle behavior. Traffic now behaves in a more realistic manner at intersections. Vehicle turning is now improved to adapt much better at high speeds. Collision detection at high speeds is enhanced to garner more realistic rapid traffic behavior. 
+
+### User defined paths in traffic manager
+
+The traffic manager has been extended with some great new functionality to extend user control over the behavior of vehicles in the traffic manager. The API now exposes new traffic manager functions that can be used to guide vehicles with a user-defined custom path.
+
+The traffic manager now exposes the `get_next_action` function to access the next action that a traffic manager guided vehicle will take and the `get_all_actions` enables the user to query all possible actions a traffic manager guided vehicle has available.
+
+The `set_path` function allows a custom route to be defined using coordinates defined as CARLA Locations:
+
+```py
+path = [carla.Location(x=-506.696198, y=179.384308, z=0.038194),
+                    carla.Location(x=-504.745972, y=232.868927, z=0.039417)]
+traffic_manager.set_path(vehicle, path)
+```
+The `set_route` function allows a custom route to be defined using route commands:
+
+```py
+route = ["Right", "Straight", "Right", "Right", "Left", "Right"]
+ traffic_manager.set_route(vehicle, route)
+```
+
+![set_route](/img/posts/2021-11-07/follow_route.gif){:class="img-fluid rounded mx-auto d-block"}
+
+### Traffic manager controlled vehicle lights
+
+The traffic manager now has functionality to control vehicle lights. Headlights, fog lights and indicators are controlled according to vehicle route data and weather conditions. Indicator lights will activate according to the next planned turn in the route, braking lights will activate when the vehicle brakes are applied and fog lights and main beams are activated according to the weather and light conditions. During night, the vehicle main beams are activated and in foggy conditions the rear fog lights are activated. 
+
+
+
+### Improved vehicle behavior - gears and braking
```

Comment:
For me, this should be its own category, as it isn't related to the Traffic Manager. All vehicles have had their physics improved, not just the ones controlled by the TM.